Khan/perseus
# View a locked line

## Summary:
Show lines on on the interactive graph widget when they're
passed in via the `lockedFigures` prop.

Issue: https://khanacademy.atlassian.net/browse/LEMS-1904

## Test plan:
`yarn jest packages/perseus/src/widgets/__tests__/interactive-graph.test.ts`

Storybook
- Go to http://localhost:6006/?path=/story/perseus-widgets-interactive-graph--segment-with-mafs-and-locked-lines
- Confirm that a green segment and a purple line show up with a close left point and an open right point

<img width="541" alt="Screenshot 2024-04-17 at 10 07 03 AM" src="https://github.com/Khan/perseus/assets/13231763/22114813-c7d7-44ad-a173-742dd12481ac">

NOTE: The dashed line segments are weird, but this will be fixed in a future PR

<img width="555" alt="Screenshot 2024-04-17 at 10 36 15 AM" src="https://github.com/Khan/perseus/assets/13231763/d1f5dc69-338e-4297-b1ac-76ca23b4e30b">

<img width="543" alt="Screenshot 2024-04-17 at 10 36 19 AM" src="https://github.com/Khan/perseus/assets/13231763/75fdafcd-d526-4e4c-9f49-41f52544e884">
